src/scripts/main.js

```diff
@@ -1,7 +1,38 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const block = document.createElement('div');
+  const header = document.createElement('h2');
+  const text = document.createElement('p');
+
+  block.setAttribute('class', `nonfication_${type}`);
+  header.setAttribute('class', title);
+  text.textContent = description;
+
+  nonfication(block, type, posTop, posRight);
+
+  document.body.append(block);
+  block.append(header, text);
+
+  setTimeout(() => {
+    block.remove();
```

Comment:
Don't need to remove element, use property visibility or display for hide element